Competition2017/src/competition/subsystems/shooter_wheel/commands/TrimShooterWheelCommand.java

```diff
@@ -10,19 +10,20 @@
 
     final DoubleProperty trimAmount;
     final ShooterWheelSubsystem shooterWheelSubsystem;
+    
     private TrimDirection direction;
     
+    public enum TrimDirection {
```

Comment:
Ideally I think this should be at the very top of the class, above the other instance variables